sygnal/apnspushkin.py

```diff
@@ -109,7 +110,7 @@ def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]) -> None:
                 nonunderstood,
             )
 
-        platform = self.get_config("platform", str)
+        platform = get_key(self.cfg, "platform", str)
```

Comment:
Was this intentional?

Reply:
Nope, totally forgot to remove that. I was considering refactoring, see my question here: https://github.com/matrix-org/sygnal/pull/264#discussion_r745050336, then thought I should probably ask before diving into that and then obviously forgot to remove that line.